### PR TITLE
feat: Add approval process create API + example

### PR DESCRIPTION
**Pull request by MCarlomagno**

### CODE REVIEW

**Review by zeljkoX — APPROVED** [review]

File: packages/approval-process/src/models/approval-process.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  network?: Network;
+  via?: string;
+  viaType?:
+    | 'EOA'
+    | 'Contract'
+    | 'Multisig'
+    | 'Gnosis Safe'
+    | 'Safe'
+    | 'Gnosis Multisig'
+    | 'Relayer'
```
Comment (shahnami) [inline packages/approval-process/src/models/approval-process.ts:54]: Note for relayer groups we need to add "Relayer Group" type here.

File: packages/approval-process/src/models/approval-process.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    | 'Unknown'
+    | 'Timelock Controller'
+    | 'ERC20'
+    | 'Governor'
+    | 'Fireblocks';
+  timelock?: Timelock;
+  multisigSender?: string;
+  fireblocks?: FireblocksProposalParams;
+  relayerId?: string;
```
Comment (shahnami) [inline packages/approval-process/src/models/approval-process.ts:63]: and also `relayerGroupId`

File: packages/approval-process/src/models/approval-process.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  via?: string;
+  viaType?:
+    | 'EOA'
+    | 'Contract'
+    | 'Multisig'
+    | 'Gnosis Safe'
+    | 'Safe'
+    | 'Gnosis Multisig'
+    | 'Relayer'
+    | 'Unknown'
```
Comment (shahnami) [inline packages/approval-process/src/models/approval-process.ts:28]:
```suggestion
    | 'Unknown'
    | 'Relayer Group'
```

File: packages/approval-process/src/models/approval-process.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    | 'Timelock Controller'
+    | 'ERC20'
+    | 'Governor'
+    | 'Fireblocks';
+  timelock?: Timelock;
+  multisigSender?: string;
+  fireblocks?: FireblocksProposalParams;
+  relayerId?: string;
```
Comment (shahnami) [inline packages/approval-process/src/models/approval-process.ts:36]:
```suggestion
  relayerId?: string;
  relayerGroupId?: string;
```

**Review by shahnami — APPROVED** [review]

**Review by shahnami — APPROVED** [review]

**Review by zeljkoX — APPROVED** [review]
LGTM

Thanks

**Review by LuisUrrutia — APPROVED** [review]

**Pull request merged**